packages/react-dom/src/client/ReactDOMEventHandle.js

```diff
@@ -140,110 +199,39 @@ export function createEventHandle(
       priority = getEventPriorityForListenerSystem(topLevelType);
     }
 
-    const listeners = new Map();
-
-    const destroy = (target: EventTarget | ReactScopeInstance): void => {
-      const listener = listeners.get(target);
-      if (listener !== undefined) {
-        listeners.delete(target);
-        const targetListeners = getEventHandlerListeners(target);
-        if (targetListeners !== null) {
-          targetListeners.delete(listener);
-        }
+    const registeredReactDOMEvents = new PossiblyWeakSet();
```

Comment:
Does this mean we leak memory on IE11? It looks like we always add targets to this Set, but it's never cleared (since it's assumed to be a WeakSet in modern browsers).

Reply:
Fixed in https://github.com/facebook/react/pull/19686.